apps/studio/components/interfaces/Connect/Connect.constants.ts

```diff
@@ -256,6 +256,20 @@ export const MOBILES: ConnectionType[] = [
       },
     ],
   },
+  {
+    key: 'exporeactnativesocialauth',
```

Comment:
I think I'd prefer to leave this out of the Studio connect modal. That's mean to be a quick connect sort of thing. 

This link though could definitely mention the social auth approach
https://share.cleanshot.com/8Pqd16hf